lib/matplotlib/axes/_base.py

```diff
@@ -2917,12 +2917,12 @@ def set_xscale(self, value, **kwargs):
         """
         Set the x-axis scale
 
-        Set the scaling of the x-axis: %(scale)s
-
-        ACCEPTS: [%(scale)s]
+        Parameters
+        ----------
+        value: float
```

Comment:
I don't think this is true at all; it accepts the _strings_ defined [here](https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/scale.py#L504).